src/platform/site-wide/wc-loader.js

```diff
@@ -1,3 +1,4 @@
+import 'core-js/features/promise';
```

Comment:
:thinking: Interesting. What's the difference between this and [`es.promise`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/polyfills/preESModulesPolyfills.js#L105)

Reply:
I think that this is the polyfill that is coming from the `static-pages` entry.

https://github.com/department-of-veterans-affairs/vets-website/blob/a532223fea1a2ec9737f9eea7b089604dd0a7a56/src/applications/static-pages/static-pages-entry.js#L1

^ That leads to:

https://github.com/department-of-veterans-affairs/vets-website/blob/a532223fea1a2ec9737f9eea7b089604dd0a7a56/src/platform/polyfills/index.js#L6

And the [docs for `core-js`](https://github.com/zloirock/core-js#ecmascript-promise) say that this is how you import _just_ the Promise polyfill:

> `core-js(-pure)/es|stable|features/promise`

Reply:
Also, from [a different section in the `core-js` docs](https://github.com/zloirock/core-js#caveats-when-using-commonjs-api):

> `modules` path is an internal API, does not inject all required dependencies and can be changed in minor or patch releases. Use it only for a custom build and/or if you know what are you doing.

So that `modules/es.promise` shouldn't be like that, if I'm understanding things correctly.